models/rke-machine.cattle.io.digitaloceanmachinetemplate.js

```diff
@@ -1,6 +1,6 @@
 import MachineTemplate from './rke-machine.cattle.io.machinetemplate';
 
-export default class DigialoceanMachineTemplate extends MachineTemplate {
+export default class DigitaloceanMachineTemplate extends MachineTemplate {
```

Comment:
All of the name dependent stuff for machine templates happens with the file name rather than class name, so from a code perspective it's safe either way. From a product perspective it's definitely `DigitalOcean`.